components/admin_console/billing/billing_subscriptions/limits.scss

```diff
@@ -0,0 +1,65 @@
+@import 'utils/variables';
+@import './mixins';
+
+.ProductLimitsPanel {
+    @include cloud-card;
+
+    &--left-panel {
+        max-width: 568px;
+        padding: 24px 32px;
+    }
+
+    &--single-panel {
+        padding: 24px calc(100% - 536px) 24px 32px;
```

Comment:
since you are making a substract in the calc function, could you please add a comment that specifies what each value refer to? or even place the value in a variable with a descriptive name so it is auto-documented